python_modules/libraries/dagster-embedded-elt/dagster_embedded_elt_tests/dlt_tests/test_dlt.py

```diff
@@ -146,3 +146,11 @@ def example_pipeline_assets(
         for mat in asset_materializations
     ]
     assert found_asset_keys == [AssetKey(["dlt_pipeline_repo_issues"])]
+
+
+def test_subset_pipeline_using_with_resources(dlt_pipeline: Pipeline) -> None:
+    @dlt_assets(dlt_source=pipeline().with_resources("repos"), dlt_pipeline=dlt_pipeline)
+    def example_pipeline_assets() -> None: ...
```

Comment:
Can we also execute the pipeline and assert that the materializations are as expected?

Reply:
Good call, added in https://github.com/dagster-io/dagster/pull/21309/commits/41602efacc61f27c63370c3ff0ca8d2f28aa7e3c, and identified a spot where I was missing the `.selected_resources` update. Thank you, @rexledesma !